### PR TITLE
fix(#3): ensure assets & scss folders exist in lib

### LINKED ISSUE #3: [Bug] Missing `assets`, `scss` folders in published package on npm
### Missing `assets` and `scss` in the new `vocabulary-styles` package

image

image

### Needed in certain places in chooser
- https://github.com/creativecommons/chooser/blob/main/src/App.vue#L175
- https://github.com/creativecommons/chooser/blob/main/src/Vocabulary/VInput.vue#L138

## Reproduction

1. clone repo and `npm install`
2. `npm run build`
3. `npm publish --dry-run` image

4. Missing `assets`, `scss` despite being present in `dist` image

## Expectation
package should contain `assets`, `scss` directories

## Environment

 - Device: Laptop
 - OS: Endeavour OS Linux 5.15.12-zen1-1-zen

### Possible Fix
- update `"files"` https://github.com/creativecommons/vocabulary-styles/blob/main/package.json#L84 in this file, to include the entire dist folder
  - drawback, would need `/dist/` attached to path for imports (not sure)
- [ ] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by sadn1ck**
Signed-off-by: Anik Das <anikdas0811@gmail.com>

## Fixes
Fixes #3 by @sadn1ck

cc: @brylie 

## Description
Makes sure `assets` and `scss` folders make their way to the final npm module as they are in use

## Tests
- `npm publish --dry-run` to make sure tarball has the folders

## Screenshots
<details>
<summary>Dry Run Logs</summary>

```
vocabulary-styles on  fix/package-contents is 📦 v0.1.0 via  v14.18.1 took 3s 
❯ npm publish --dry-run
npm notice 
npm notice 📦  @creativecommons/vocabulary-styles@0.1.0
npm notice === Tarball Contents === 
npm notice 1.1kB   LICENSE                                            
npm notice 220.5kB dist/css/vocabulary-styles.css                     
npm notice 1.6MB   dist/assets/demo/broadway_tower.jpg                
npm notice 77.1kB  dist/assets/cclogo-shared-image.jpg                
npm notice 89.3kB  dist/assets/demo/lion_dafrique.jpg                 
npm notice 4.0kB   dist/js/vocabulary-styles.js                       
npm notice 2.9kB   package.json                                       
npm notice 806B    CODE_OF_CONDUCT.md                                 
npm notice 1.4kB   CONTRIBUTING.md                                    
npm notice 339B    README.md                                          
npm notice 9.4kB   dist/assets/icons/favicon.png                      
npm notice 19.1kB  dist/assets/demo/left_index.png                    
npm notice 40.5kB  dist/assets/demo/show_code.png                     
npm notice 103.8kB dist/assets/demo/tab_content.png                   
npm notice 506.4kB dist/assets/titlecard.png                          
npm notice 500B    dist/scss/breadcrumb.scss                          
npm notice 5.0kB   dist/scss/button.scss                              
npm notice 12.6kB  dist/scss/card.scss                                
npm notice 858B    dist/scss/color_overrides.scss                     
npm notice 1.8kB   dist/scss/color.scss                               
npm notice 2.1kB   dist/scss/footer.scss                              
npm notice 4.9kB   dist/scss/form.scss                                
npm notice 449B    dist/scss/github_corner.scss                       
npm notice 9.2kB   dist/scss/global_header.scss                       
npm notice 7.8kB   dist/scss/header.scss                              
npm notice 1.5kB   dist/scss/image.scss                               
npm notice 115B    dist/scss/layout.scss                              
npm notice 1.7kB   dist/scss/locale.scss                              
npm notice 1.1kB   dist/scss/modal.scss                               
npm notice 2.2kB   dist/scss/notification.scss                        
npm notice 1.5kB   dist/scss/pagination.scss                          
npm notice 205B    dist/scss/profile_image.scss                       
npm notice 925B    dist/scss/sidebar_menu.scss                        
npm notice 491B    dist/scss/spacing_overrides.scss                   
npm notice 1.3kB   dist/scss/spacing.scss                             
npm notice 510B    dist/scss/table_of_contents.scss                   
npm notice 1.7kB   dist/scss/table_of_progress.scss                   
npm notice 1.6kB   dist/scss/table.scss                               
npm notice 1.7kB   dist/scss/tabs.scss                                
npm notice 358B    dist/scss/typography_overrides.scss                
npm notice 2.9kB   dist/scss/typography.scss                          
npm notice 1.1kB   dist/scss/vocabulary.scss                          
npm notice 10.4kB  dist/assets/license_badges/big/by_nc_nd.eu.svg     
npm notice 10.3kB  dist/assets/license_badges/big/by_nc_nd.svg        
npm notice 5.5kB   dist/assets/license_badges/small/by_nc_nd.svg      
npm notice 12.4kB  dist/assets/license_badges/big/by_nc_sa.eu.svg     
npm notice 12.3kB  dist/assets/license_badges/big/by_nc_sa.svg        
npm notice 4.8kB   dist/assets/license_badges/small/by_nc_sa.svg      
npm notice 8.7kB   dist/assets/license_badges/big/by_nc.eu.svg        
npm notice 9.6kB   dist/assets/license_badges/big/by_nc.svg           
npm notice 4.7kB   dist/assets/license_badges/small/by_nc.svg         
npm notice 7.5kB   dist/assets/license_badges/big/by_nd.svg           
npm notice 4.4kB   dist/assets/license_badges/small/by_nd.svg         
npm notice 9.5kB   dist/assets/license_badges/big/by_sa.svg           
npm notice 4.8kB   dist/assets/license_badges/small/by_sa.svg         
npm notice 5.1kB   dist/assets/license_badges/big/by.svg              
npm notice 3.7kB   dist/assets/license_badges/small/by.svg            
npm notice 6.3kB   dist/assets/license_badges/big/cc_zero.svg         
npm notice 4.6kB   dist/assets/license_badges/small/cc_zero.svg       
npm notice 7.8kB   dist/assets/logos/products/certificate.svg         
npm notice 7.9kB   dist/assets/logos/products/certificates.svg        
npm notice 6.6kB   dist/assets/logos/products/chooser.svg             
npm notice 1.4kB   dist/assets/github_corner.svg                      
npm notice 8.7kB   dist/assets/logos/products/global_network.svg      
npm notice 9.4kB   dist/assets/logos/products/global_summit.svg       
npm notice 11.8kB  dist/assets/logos/products/legal_database.svg      
npm notice 2.5kB   dist/assets/logos/cc/letterheart.svg               
npm notice 3.2kB   dist/assets/logos/cc/lettermark.svg                
npm notice 6.9kB   dist/assets/logos/cc/logomark.svg                  
npm notice 8.7kB   dist/assets/logos/products/open_source.svg         
npm notice 5.7kB   dist/assets/license_badges/big/publicdomain.svg    
npm notice 4.1kB   dist/assets/license_badges/small/publicdomain.svg  
npm notice 1.5kB   dist/assets/puff.svg                               
npm notice 6.3kB   dist/assets/logos/products/search.svg              
npm notice 13.7kB  dist/assets/logos/products/state_of_the_commons.svg
npm notice 7.4kB   dist/assets/logos/products/vocabulary.svg          
npm notice === Tarball Details === 
npm notice name:          @creativecommons/vocabulary-styles      
npm notice version:       0.1.0                                   
npm notice package size:  2.5 MB                                  
npm notice unpacked size: 3.0 MB                                  
npm notice shasum:        3edcd16149285ad4a5d54e8970bb2fa72d0156e6
npm notice integrity:     sha512-sDTnA9k/byfZI[...]sScKoDSOPudOg==
npm notice total files:   76                                      
npm notice 
+ @creativecommons/vocabulary-styles@0.1.0
```
</details>

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,6 +85,8 @@
     "CODE_OF_CONDUCT.md",
     "CONTRIBUTING.md",
     "dist/css/",
-    "dist/js/"
+    "dist/js/",
+    "dist/scss",
+    "dist/assets"
```
Comment (brylie) [inline package.json]: If the goal is to include the `scss` files, we probably don't need to have the `assets`. From what I understand, Storybook uses the `assets`, which include large image files (e.g., broadway_tower.jpg) that aren't essential to this project.
```suggestion
```

**Review by brylie — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (sadn1ck) [reply package.json]: There is just [one place here](https://github.com/creativecommons/chooser/blob/main/src/components/FooterSection.vue#L16) where an svg is directly imported from the vocabulary package, which is why I included it.

Should I go ahead and remove the `assets` folder anyway? @brylie

Comment (brylie) [reply package.json]: Hmm, well, we can include the `assets/logos/` perhaps. However, I want to keep this package slim and avoid packaging documentation dependencies.

We can explore where our logo assets should reside as part of a later release.

Comment (sadn1ck) [reply package.json]: Updated to have just `logos`, we can update stuff later yep!

**Review by brylie — APPROVED** [review]

**Pull request merged**